org.jacoco.core/src/org/jacoco/core/instr/Instrumenter.java

```diff
@@ -114,7 +114,7 @@ protected String getCommonSuperClass(final String type1,
 				return instrument(new ClassReader(buffer));
 			}
 		} catch (final RuntimeException e) {
-			throw instrumentError(name, e);
+			throw instrumentError("class " + name, e);
```

Comment:
@marchof just to clarify: so accepting that agent will generate message without word "class" ?